test/fixtures/settings.yml

```diff
@@ -384,3 +384,8 @@ attribute80:
   category: Setting::Auth
   default: 30
   description: "Foreman will block user login after this number of failed login attempts for 5 minutes from offending IP address. Set to 0 to disable bruteforce protection"
+attribute80:
+  name: bcrypt_cost
+  category: Setting::Auth
+  default: 10
```

Comment:
why 10, isn't it a bit on the lower side (if it's between 5-30)?

Reply:
If you set this to 30, test suite will not finish in an hour on desktop CPU 2017 (I cancelled it), I think we are on safe side :-)

Reply:
After reading some material it looks like 10 should be safe but setting it to 11-12 should not be a problem for todays computers (hashing up to 300ms in the worst case). I think I am gonna increase this to 11 or 12 and I will add telemetry histogram which will report hashing duration so admin can decide herself.

https://security.stackexchange.com/questions/17207/recommended-of-rounds-for-bcrypt

Reply:
> If you set this to 30, test suite will not finish in an hour on desktop CPU 2017 (I cancelled it), I think we are on safe side :-)

I know that 30 is expensive, thanks for explaining the "10" part.